### PR TITLE
Fix/remove old links

### PR DESCRIPTION
**Pull request by Zed314**
Removed references from old project

Removed references from old project, kept link to documentation.
Removed unused privacy tab

**Checklist**

- [X] The web version was tested and it is running ok
- [X] The desktop version was tested and it is running ok
- [X] The release version was updated on package.json files

### CODE REVIEW

File: packages/theme/src/i18n/en/openDialog.ts
Comment (laisspportugal) [inline packages/theme/src/i18n/en/openDialog.ts:1]: We actually need to change everything here to adapt for the new product. Thanks for the hint

File: packages/theme/src/i18n/en/openDialog.ts [hunk 1]
```diff
@@ -32,5 +32,5 @@ export const openDialog = {
   startCollaboratingDescription:
     "Make the most of your Foxglove account – whether you want to dive deep on your data or share tools with your teammates.",
```
Comment (laisspportugal) [inline packages/theme/src/i18n/en/openDialog.ts:33]: Can you please change to
`"Make the most of your Foxbox account – whether you want to dive deep on your data or share tools with your teammates."` ?

File: packages/studio-base/src/i18n/en/openDialog.ts [hunk 2]
```diff
@@ -32,5 +32,5 @@ export const openDialog = {
   startCollaboratingDescription:
     "Make the most of your Foxglove account – whether you want to dive deep on your data or share tools with your teammates.",
   uploadToDataPlatform: "Upload to Data Platform",
-  viewOurDocs: "View our docs",
```
Comment (laisspportugal) [inline packages/studio-base/src/i18n/en/openDialog.ts:35]: We would like to keep it that way until we build our own documentation

File: packages/studio-base/src/AppSetting.ts [hunk 3]
```diff
@@ -16,7 +16,6 @@ export enum AppSetting {
   ENABLE_NEW_TOPNAV = "enableNewTopNav",
 
   // Privacy
-  TELEMETRY_ENABLED = "telemetry.telemetryEnabled",
```
Comment (laisspportugal) [inline packages/studio-base/src/AppSetting.ts:19]: We would like to understand the reason why youre removing telemetry. Can you please elaborate?

File: packages/studio-desktop/src/main/StudioWindow.ts [hunk 4]
```diff
@@ -85,7 +85,7 @@ function newStudioWindow(deepLinks: string[] = [], reloadMainWindow: () => void)
       nodeIntegration: false,
       additionalArguments: [
         `--allowCrashReporting=${crashReportingEnabled ? "1" : "0"}`,
-        `--allowTelemetry=${telemetryEnabled ? "1" : "0"}`,
```
Comment (laisspportugal) [inline packages/studio-desktop/src/main/StudioWindow.ts:88]: Same as the previous comment

File: packages/studio-base/src/AppSetting.ts (hunk 3 above)
Comment (aneuwald-ctw) [inline packages/studio-base/src/AppSetting.ts:19]: Can you explain why did you decide to remove telemetry?

File: packages/studio-desktop/src/main/telemetry.ts
Comment (aneuwald-ctw) [inline packages/studio-desktop/src/main/telemetry.ts:1]: Also here, why remove the "telemetryEnabled", but no all the functionality of getTelemetrySettings

**Review by aneuwald-ctw — CHANGES_REQUESTED** [review]

**Comment by laisspportugal** [comment]
Thank you for your contribution, we really appretiate that. We would like to understand a few things. Please see the comments.

File: packages/studio-base/src/AppSetting.ts (hunk 3 above)
Comment (Zed314) [reply packages/studio-base/src/AppSetting.ts:19]: I assumed that telemetry was already removed, as I was not able to find code that do the telemetry.

Comment (Zed314) [reply packages/studio-base/src/AppSetting.ts:19]: I am putting it back :-)

File: packages/theme/src/i18n/en/openDialog.ts [hunk 5]
```diff
@@ -32,5 +32,5 @@ export const openDialog = {
...
   uploadToDataPlatform: "Upload to Data Platform",
-  viewOurDocs: "View our docs",
+  viewOurDocs: "View Foxglove docs",
```
[the remark this answers is not in the document]
Comment (Zed314) [reply packages/theme/src/i18n/en/openDialog.ts:35]: Alright, I reverted it.

File: packages/studio-desktop/src/main/StudioWindow.ts (hunk 4 above)
Comment (Zed314) [reply packages/studio-desktop/src/main/StudioWindow.ts]: Alright, I reverted it.

File: packages/studio-base/src/AppSetting.ts (hunk 3 above)
Comment (Zed314) [reply packages/studio-base/src/AppSetting.ts]: I think it was a mistake, as I thought it was not used anymore. 
I reverted it.

**Review by laisspportugal — DISMISSED** [review]

**Review by aneuwald-ctw — APPROVED** [review]

**Comment by laisspportugal** [comment]
@Zed314 there is a remaining lint issue, please verify it

**Comment by aneuwald-ctw** [comment]
@Zed314 could you please fix the lint problems? Then we can finally merge and close this PR :)

**Comment by laisspportugal** [comment]
This pr will be closed since we are doing the suggested changes in another PR: https://github.com/bmw-software-engineering/lichtblick/pull/116 


Thank you for your contribution

**Pull request closed without merging**